.travis.yml

```diff
@@ -107,4 +107,5 @@ script:
   - make clean
   - make build_all PLATFORM=x86 DEFINE="${DEFINES}"
   - make test_all PLATFORM=x86 DEFINE="${DEFINES}"
+  - make test_all_python
```

Comment:
Travis is failing because it can't find various requirements from the libraries - I'd suggest fixing this by adding something in the `before_script` section which finds all `requirements.txt` files in the repo (they contain python requirements for each project) and runs `pip install -r <path/to/requirements.txt>` on them.